charts/zookeeper/values.yaml

```diff
@@ -29,7 +30,10 @@ pod:
   # tolerations: []
   # env: []
   # annotations: {}
-  # securityContext: {}
+  # securityContext:
+    # Mention runAsUser as auto in case of openshift
```

Comment:
I would lose the auto keyword (sorry I suggested that), and keep it simple:  Either specify a value for runAsUser, which can be 0 or any number, or don't specify a value, in which case it will be determined automatically.  (which is valid in non openshift env. also.  But is especially useful in Openshift envs)

Reply:
Removed `auto`